src/chrome/content/rules/M-net.de.xml

```diff
@@ -1,9 +1,82 @@
 <ruleset name="M-net Telekommunikations GmbH" default_off="broken">
```

Comment:
At the tine the request was made the mixed content broke the page and a few places had redirects to plain. I will check it again.